facia-press/app/frontpress/PressedCollectionDeduplication.scala

```diff
@@ -51,7 +57,10 @@ object PressedCollectionDeduplication {
       collectionV: PressedCollectionVisibility,
   ): PressedCollectionVisibility = {
     // Essentially deduplicate the backfill of collectionV using header values values from accum's elements curated and backfill
-    val accumulatedHeaderURLsForDeduplication: Seq[String] = getHeaderURLsFromCuratedAndBackfilledAtDepth(accum, 10)
+    val accumulatedHeaderURLsForDeduplication: Seq[String] =
+      getHeaderURLsFromCuratedAndBackfilledAtDepth(accum, 10) ++ getHeaderURLsFromTrailsList(
```

Comment:
Do you know why we're taking 10?

Reply:
I'm not actually sure, although my guess would be that it's because a collection can have way more backfill cards than would ever be displayed in the main container, but we don't want to remove a card from later containers just because it's in the  'show more' section of a container above. With 10 being an approximation of the max number of cards that would be displayed in the main container?

If this is the purpose then I suspect that we could refactor it, but I thought I should keep this PR targeted and maybe come back to some of these  other  issues later on?

The comments  in the file hint at a  planned  refactor  of the  visibility  logic as well, which doesn't look like it was  every  finished, so it  could be good to see if we can find  any more context on that?

Reply:
Yeah absolutely not in the scope of this PR! You've done a great job already at making things simpler!